### DISCUSSION TITLE
Create an user

### DISCUSSION THREAD

**Discussion by starnakin** [louislam/dockge#440] [Ask for Help]
Can I add new user to my dockge instance ? Thanks

**Comment by youssefshibl** [comment]
While trying to explore the code if found that after making the initial user, you can't make another user, and if you forget the password of user and want to make new user you should delete data , why can't make new user or why not in creating new user delete old user and make new one ?
![Screenshot from 2024-03-19 21-33-44](https://github.com/louislam/dockge/assets/63800183/15e8ffbe-b1fb-4005-81eb-0f9c68633d9a)
